sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3486,7 +3486,7 @@ def hpat_pandas_series_min(self, axis=None, skipna=None, level=None, numeric_onl
 
     Limitations
     -----------
-    - Parameters level, numeric_only are currently unsupported by Intel Scalable Dataframe Compiler
+    Parameters ``level``, ``numeric_only``, ``axis`` are supported only with default value ``None``.
```

Comment:
We should say that this parameters should not be used (should be omitted) or should be in default values. 
```suggestion
    Parameters ``level``, ``numeric_only``, ``axis`` are currently unsupported by Intel Scalable Dataframe Compiler.
```